wordpress-mobile/WordPress-Android
# Fix inflation issue on lenovo

Fixes #9989 and https://github.com/wordpress-mobile/WordPress-Android/issues/10168

It seems some styles are missing on Lenovo devices with Android 5.x. I don't have any such device, so I can't test the changes. But I'd give it a try.

To test:
1. Open reader
2. Make sure the currently selected filter looks ok 
3. Open the filter drop down and make sure it looks ok
-----
1. Open My site
2. Comments
3. Make sure the currently selected filter looks ok 
4. Open the filter drop down and make sure it looks ok


Update release notes:

- The changes are too minor
